### ISSUE TITLE
Loading GroundOverlays and KmlOverlays from file://

### ISSUE THREAD

**Issue by MichaelGooden** [wf9a5m75/phonegap-googlemaps-plugin#405]
I see that support for loading marker images from file:// paths was added recently, however for ground overlays and kml overlays we can currently only load files from an http:// path or files that exists in the app's assets directory.

Could you please add support for loading GroundOverlays and KmlOverlays from a file:// URI?

**Comment by wf9a5m75** [comment]
Yes, I'm planing to do this in the next version, but my private time is really less these days. Please wait for a while.

**Comment by wf9a5m75** [comment]
This feature request has implemented into v1.2.5

You can specify the image file path using `file://`, `cdvfile://`, relative path or absolute file path.

```js
map.addKmlOverlay({
    'url': "kml/polygon.kml"
}, function(kmlOverlay) {});
```

```js
map.addMarker({
    'icon': "images/marker.png"
   ....
}, function(marker) {});
```

```js
map.addGroundOverlay({
    'icon': "file:///...../myGroundOverlay.png"
   ....
}, function(marker) {});
```

**Comment by wf9a5m75** [comment]
Oh, I let you know that loading GroundOverlay has not implemented yet.
